UefiPayloadPkg/UefiPayloadEntry/UniversalPayloadEntry.inf

```diff
@@ -69,14 +62,12 @@
   gUniversalPayloadAcpiTableGuid
   gUniversalPayloadPciRootBridgeInfoGuid
   gUniversalPayloadSmbios3TableGuid
-
+  gUniversalPayloadBaseGuid
+  gUniversalPayloadDeviceTreeGuid

```

Comment:
It looks in this commit these 2 GUIDs are not used in ELF universal payload. Please double check if they are really needed.

Reply:
Will remove these 2 GUID.

Reply:
resolved